mindmeld/components/dialogue.py

```diff
@@ -563,6 +564,7 @@ def _decorator(func):
             # Support syntax: @middleware()
             return _decorator
 
+        #TODO: The code below is unreachable
```

Comment:
Maybe @minhtuev can confirm, but I think we should just remove those lines. They were added with the original DialogueFlow PR, and didn't do anything there either.
https://github.com/cisco/mindmeld/commit/bbfff15da31d1bc4d2d4cc1b866bb26cf5be67ad

Reply:
Let's just remove the code below

Reply:
done